plugins/modules/mso_schema_site_vrf_region_hub_network.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Dag Wieers (@dagwieers) <dag@wieers.com>
+# Copyright: (c) 2020, Lionel Hercot (@lhercot) <lhercot@cisco.com>
+# Copyright: (c) 2020, Cindy Zhao (@cizhao) <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: mso_schema_site_vrf_region_hub_network
+short_description: Manage site-local VRF region hub network in schema template
+description:
+- Manage site-local VRF region hub network in schema template on Cisco ACI Multi-Site.
+author:
+- Dag Wieers (@dagwieers)
+- Lionel Hercot (@lhercot)
+- Cindy Zhao (@cizhao)
+options:
+  schema:
+    description:
+    - The name of the schema.
+    type: str
+    required: yes
+  site:
+    description:
+    - The name of the site.
+    type: str
+    required: yes
+  template:
+    description:
+    - The name of the template.
+    type: str
+    required: yes
+  vrf:
+    description:
+    - The name of the VRF.
+    type: str
+  region:
+    description:
+    - The name of the region.
+    type: str
+  hub_network:
+    description:
+    - Whether hub network is enabled or not in this region.
+    type: bool
+    default: false
+  hub_network_name:
+    description:
+    - The name of the hub network to manage.
+    type: str
+    choices: [ hub-default, hub-test ]
+  tenant:
```

Comment:
I am not sure if it is clear enough with this way of putting the tenant. Because this tenant relates to the hub_network itself. So I am thinking that doing something like:

```
hub_network:
  name:
  tenant:
```

Might be better. What do you think?